heron-adapter/src/main/java/org/openconnectors/pulsar/HeronPulsarSink.java

```diff
@@ -0,0 +1,41 @@
+package org.openconnectors.pulsar;
+
+import com.twitter.heron.streamlet.Context;
+import com.twitter.heron.streamlet.Sink;
+import org.openconnectors.PulsarSink;
+import org.openconnectors.config.ConfigProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+
+public class HeronPulsarSink implements Sink<byte[]> {
```

Comment:
@merlimat Whats the story around typed Pulsar client? For now it looks ok, but can we have pulsar typed clients so that marshalling details are taken care of?

Reply:
@jerrypeng Does it make sense to take a general type rather than just bytes and convert the type to bytes

Reply:
@srkukarni the type would have to be serializable.  We haven't discussed our serialization strategy for connectors.  We currently does have an interface for users to specify serializers.